README.md

```diff
@@ -1,70 +1,66 @@
-# Getting Started with Create React App
+- ![](https://img.shields.io/badge/Microverse-blueviolet)
 
-This project was bootstrapped with [Create React App](https://github.com/facebook/create-react-app).
+# Math Magicians
+
+ Description the project.
 
```

Comment:
- [ ] As required by the previous reviewer you should customize this file and add the necessary information I suggest you add a short description of the project then users can have an idea about the project 🙃 🙃